File: backend/src/main/java/be/technobel/parsemaster/form/ChangePasswordForm.java
package be.technobel.parsemaster.form;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record ChangePasswordForm(
  @Schema(maxLength = 255, minLength = 1)
  @Size(max = 255, message = "length>255")
  @NotBlank(message = "blank")
  String password
) {}

File: src/main/kotlin/be/technobel/playzone/pl/models/forms/ChangePasswordForm.kt
package be.technobel.playzone.pl.models.forms

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull

data class ChangePasswordForm (
	@field:NotNull(message = "required")
	@field:NotBlank(message = "not_blank")
	val password: String? = null,
)